.gitignore

```diff
@@ -1,4 +1,5 @@
 node_modules
+.DS_Store
```

Comment:
Could you remove this addition? I don't think we've seen this file in PRs at all so it doesn't seem necessary.

Reply:
I just had one slip into this PR (which is why I rebased and added it). It's a meta data on MacOS and it's generated automatically when you view a directory. I was surprised you didn't have it.

Reply:
Personally I have it in `~/.gitignore`. I'd just rather not balloon `.gitignore` with OS level stuff.

Reply:
Coo cool, no prob. I'll remove it.

Reply:
Only things related to the project should be in the project gitignore. The rest should be in your own global gitignore. 

https://gist.github.com/subfuzion/db7f57fff2fb6998a16c